default_theme/index.js

```diff
@@ -71,21 +71,20 @@ module.exports = function (comments, options, callback) {
       }
     }
   };
+  
+  imports.renderSectionList =  _.template(fs.readFileSync(path.join(__dirname, 'section_list._'), 'utf8'), {
```

Comment:
Running this locally produces the error `ReferenceError: imports is not defined` - it doesn't look like the imports variable is defined in this file?